### PR TITLE
Switch to device auth

### PR DESCRIPTION
**Pull request by twolaw**
[Device authentication](https://trakt.docs.apiary.io/#reference/authentication-devices) is easier to setup for users with command line scripts.
Users don't have to copy/paste a long url but only a code.
Moreover, the pytrakt function of device_auth stores the `oauth_refresh` token which can be used to automatically refresh the `oauth_token` every 3 months.

### CODE REVIEW

**Comment by glensc** [comment]
forgotten cross reference: https://github.com/Taxel/PlexTraktSync/issues/20

File: get_env_data.py [hunk 1]
```diff
@@ -21,10 +21,9 @@
     with open(env_file, "w") as txt:
         txt.write("PLEX_TOKEN=-\n")
 
-trakt.APPLICATION_ID = '65370'
-trakt.core.AUTH_METHOD=trakt.core.OAUTH_AUTH
+trakt.core.AUTH_METHOD=trakt.core.DEVICE_AUTH
 trakt_user = input("Please input your Trakt username: ")
-trakt.init(trakt_user, store=True)
+trakt.init(store=True)
```
Comment (glensc) [inline get_env_data.py]: `trakt_user` is no longer used for trakt auth. do we need to keep asking username from the user?

Comment (twolaw) [reply get_env_data.py]: You're wrong, it is still used. Read main.py

Comment (glensc) [reply get_env_data.py]: I'm not wrong, it's no longer used for trakt auth in that file. that's why asked is it not used elsewhere as well.

**Pull request merged**